kura/org.eclipse.kura.api/src/main/java/org/eclipse/kura/security/SecurityService.java

```diff
@@ -46,4 +44,24 @@ public interface SecurityService {
      */
     public boolean isDebugEnabled();
 
+    /**
+     * This method allows to apply the default production security policy available in the system.
```

Comment:
We could also mention that the changes to the security policy are persistent and should remain in effect after a framework restart